plugins.json

```diff
@@ -433,13 +433,13 @@
     },
     "Greenwing": {
         "title": "Greenwing",
-        "version": "1.3.1",
+        "version": "1.3.2",
         "author": "Confexion",
         "license": "MIT",
         "description": "Greenwing is a kanboard beautify theme with a modern design.",
         "homepage": "https://github.com/Confexion/Greenwing",
         "readme": "https://github.com/Confexion/Greenwing/blob/master/README.md",
-        "download": "https://github.com/Confexion/Greenwing/releases/download/1.3.2/Greenwing.zip",
+        "download": "https://github.com/Confexion/Greenwing/archive/refs/tags/1.3.2.zip",
```

Comment:
The zip file doesn't provide the expected folder hierarchy. See https://github.com/kanboard/website#how-to-add-a-new-plugin-to-the-list

```
$ wget https://github.com/Confexion/Greenwing/archive/refs/tags/1.3.2.zip
$ unzip 1.3.2.zip
Archive:  1.3.2.zip
eb1bb3ad8e95dcf5ae7bc8266bb60a1b57f64e60
   creating: Greenwing-1.3.2/
  inflating: Greenwing-1.3.2/ColorModel.php
  inflating: Greenwing-1.3.2/Config.php
...
```

Reply:
Thanks for pointing this out. @jbaubree Can you please add a zip file to your v1.3.2 release according to the guide @fguillot mentioned above? That'd be great. I'll update my PR here so it just bumps the version number.